fdbserver/RestoreUtil.h

```diff
@@ -34,6 +34,21 @@
 #include <cstdint>
 #include <cstdarg>
 
+#define FR_DEBUG 1
+#define FR_VERBOSE_DEBUG 1
+
+#ifdef FR_DEBUG
+#define FRTraceEvent(...) TraceEvent(SevDebugVerbose, "FastRestoreDebug", UID())
```

Comment:
I was thinking you were planning to do something like:

```#define FR_DEBUG_SEVERITY SevDebugVerbose```

But if you go this route, it looks like you are currently using the same priority if `FR_DEBUG` is enabled or not.

Reply:
Yes, Both FR_VERBOSE_DEBUG and FR_DEBUG uses the same severity level. 
Muting one is by undermining the variable.